src/helics/common/AsioContextManager.h

```diff
@@ -27,7 +27,11 @@ SPDX-License-Identifier: BSD-3-Clause
 #include <string>
 #include <vector>
 
+#define ASIO_STANDALONE 1
+#define ASIO_ERROR_CATEGORY_NOEXCEPT noexcept(true)
```

Comment:
do these only apply to the context manager, or all of ASIO?  If all of asio should we make the defines in CMAKE as part of the project, rather than defining them wherever we include ASIO?

Reply:
I’m guessing all of Asio. I’ll try with the noexcept one in CMake which already had ASIO_STANDALONE. I wasn’t sure it was actually getting defined correctly since for some reason it was trying to use BOOST_NOEXCEPT even though ASIO_STANDALONE disables using Boost.

Reply:
@phlptp It's just the context manager that needs it. I guess there's something off about the way io_context.hpp includes the ASIO header for setting up definitions correctly.